packages/util-logger/src/util-logger.ts

```diff
@@ -0,0 +1,156 @@
+// Config dependency injected into logger to break circular dependency
+let config: any = null
+export const setConfig = (_config: any) => {
+  config = _config
+}
+
+/**
+ * The levels of the logger
+ */
+export enum LEVELS {
+  debug = 5,
+  info = 4,
+  log = 3,
+  warn = 2,
+  error = 1,
+}
+
+/**
+ * Builds a message formatted for the logger
+ * @param options - The options for the log
+ * @param options.title - The title of the log
+ * @param options.message - The message of the log
+ * @returns The message formatted for the logger
+ * @example
+ * buildLoggerMessageArgs({ title: "My Title", message: "My Message" })
+ */
+const buildLoggerMessageArgs = (options: {
+  title: string
+  message: string
+}): string[] => {
+  const {title, message} = options
+  return [
+    `
+    %c${title}
+    ============================
+
+    ${message}
+
+    ============================
+    `
+      .replace(/\n[^\S\r\n]+/g, "\n")
+      .trim(),
+    "font-weight:bold;font-family:monospace;",
+  ]
+}
+
+/**
+ * Logs messages based on the level of the message and the level set in the config
+ * @param options - The options for the log
+ * @param options.title - The title of the log
+ * @param options.message - The message of the log
+ * @param options.level - The level of the log
+ * @param options.always - Whether to always show the log
+ * @example
+ * log({ title: "My Title", message: "My Message", level: LEVELS.warn, always: false })
+ */
+export const log = async (options: {
+  title: string
+  message: string
+  level: number
+  always?: boolean
+}) => {
+  const {title, message, level, always} = options
+  const configLoggerLevel = await config.get("logger.level", LEVELS.warn)
```

Comment:
It looks like something was lost here while rebasing.  The nullish case used to be handled here but now it is missing

Reply:
```suggestion
  const configLoggerLevel = (await config?.()?.get<number>("logger.level")) ?? LEVELS.warn
```